src/Application/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,17 @@
+using Microsoft.Extensions.DependencyInjection;
```

Comment:
The class of the file should correspond to the name of the file.
Better just do it like in the Core where its a file called DependencyInjection.cs for each project in the root of it rather than a folder.
This is essentially a module of the project

Reply:
0ad01fb